python3.hcl

```diff
@@ -4,7 +4,7 @@ binaries = ["install/bin/pip3", "install/bin/python3"]
 test = "python3 -m pip install flake8"
 env = {
   "PYTHONPYCACHEPREFIX": "${HERMIT_ENV}/.hermit/python/cache",
-  "PIP_CACHE_DIR": "${HERMIT_ENV}/.hermit/python/cache/pip",
```

Comment:
Unfortunately there's a bug in recursive substitution where it doesn't always work, which is why these aren't deduplicated.

Reply:
Done!